CONTRIBUTING.md

```diff
@@ -55,18 +52,7 @@ Similar rules can be applied when contributing bug fixes - it's always best to
 discuss the implementation in the bug report first if you are not 100% about
 what would be the best fix.
 
-You can refer to the [Pull request review process](https://docs.godotengine.org/en/latest/contributing/workflow/pr_review_guidelines.html)
-for insights into the intended lifecycle of pull requests. This should help you
-ensure that your pull request fulfills the requirements.
-
-In addition to the following tips, also take a look at the
-[Engine development guide](https://docs.godotengine.org/en/latest/contributing/development/index.html)
-for an introduction to developing on Godot.
-
-The [Contributing docs](https://docs.godotengine.org/en/latest/contributing/how_to_contribute.html)
-also have important information on the [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html)
-(with a helpful guide for Git usage), and our [Code style guidelines](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html)
-which all contributions need to follow.
+Visit the [Blazium Discord](https://discord.gg/redot) for information and guides related to contribution and pull requests.
```

Comment:
still has redot in it

Reply:
What is the public url for the Blazium discord? I'm happy to update this code with it :)

Reply:
work complete